h/static/scripts/group-list-controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('h').controller(
```

Comment:
I would move this line to app.coffee and just export the controller. Also, use `// @ngInject` on the constructor.